textinput.go

```diff
@@ -40,9 +40,6 @@ type keyboardShortcutsGLFW struct {
 // all hardcoded because theres not pluggable renderer system.
 var defaultTextinputPlugin = &textinputPlugin{}
 
-var _ Plugin = &textinputPlugin{}     // compile-time type check
-var _ PluginGLFW = &textinputPlugin{} // compile-time type check
```

Comment:
I think this is a useful assertion as well, textinputPlugin MUST implement PluginGLFW, otherwise it will not be recognized as such and no error will be thrown, it just silently won't work.

Reply:
Same for earlier removals of PluginGLFW compile-time type check.

Reply:
The `var _ Plugin = &textinputPlugin{}` is indeed duplicate, it's also checked in `opt = append(opt, AddPlugin(defaultTextinputPlugin))`.